.nvmrc

```diff
@@ -0,0 +1 @@
+v16.14.2
```

Comment:
I added https://github.com/shadowspawn/node-version-usage since it more widely supported format.
Please remove.